frontend/src/app/data-entry/agent/agent.component.html

```diff
@@ -0,0 +1,245 @@
+                                Is this group defined by their connection to a place?
+                            </label>
+
+                            <select class="form-select">
+                                <option selected>Yes</option>
+                                <option>No</option>
+                            </select>
+                        </div>
+                        <div class="mb-4">
+                            <label class="form-label">
+                                Select one of the places that are represented in the source.
+                            </label>
+
+                            <select class="form-select">
+                                <option selected>The Abbey of the Holy Cross</option>
+                                <option>A cave</option>
+                            </select>
+                        </div>
+                        <div class="mb-4">
+                            <label class="form-label">
+                                How is this information presented in the text?
+                            </label>
+
+                            <select class="form-select">
+                                <option>Implied</option>
+                                <option selected>Mentioned</option>
+                                <option>Described</option>
+                            </select>
+                        </div>
+                        <div class="mb-4">
+                            <label class="form-label">
+                                Additional notes
+                            </label>
+                            <textarea class="form-control" rows="2"></textarea>
+                        </div>
+                    </div>
+                </ng-template>
+            </div>
+        </div>
+    </div>
+    <div ngbAccordionItem *ngIf="agent === 1">
+        <h2 ngbAccordionHeader>
+            <button ngbAccordionButton>Connections</button>
+        </h2>
+        <div ngbAccordionCollapse>
+            <div ngbAccordionBody>
+                <ng-template>
+                    <div class="mb-4">
+                        <label class="form-label">
+                            Can we cross-reference this agent with other sources?
+                        </label>
+                        <div class="form-check">
+                            <input class="form-check-input" type="radio" checked="true">
+                            <label class="form-check-label">
+                                Yes, this person is a historical or religious figure;
+                                they may appear in other sources
+                            </label>
+                        </div>
+                        <div class="form-check">
+                            <input class="form-check-input" type="radio">
+                            <label class="form-check-label">
+                                No, this person is an unknown individual
```

Comment:
I would lean more towards 'uniquely identifiable' and 'generic' rather than 'historical'. 'A monk' is still a historical (and religious) figure, even if we do not know their name.

Reply:
Good point. I'm not sure about the best terminology here, but "identifiable" is probably the clearest term we have come up with.